src/templates/OutdatedBrowser.html

```diff
@@ -0,0 +1,9 @@
+<div class="outdated-browser">
+  <p>
+    Sorry, you are using an outdated browser.
+  </p>
+  <p>
+    Please <a href="http://outdatedbrowser.com/en">upgrade to a newer version</a>
```

Comment:
Oh yeah, good point, hrm. Yeah agree, not sure what we can do about that for the moment...